satellite/audit/verifier.go

```diff
@@ -359,6 +359,7 @@ func (verifier *Verifier) Reverify(ctx context.Context, path storj.Path) (report
 	}
 
 	pieceHashesVerified := make(map[storj.NodeID]bool)
+	pieceHashesVerifiedMutex := &sync.Mutex{}
 	defer func() {
 		// for each node in Fails and PendingAudits, remove if piece hashes not verified for that segment
```

Comment:
you need a mutex inside the defer as well due to the double send to ch

Reply:
done